packages/evolution-frontend/src/styles/survey/_base.scss

```
@@ -126,112 +158,124 @@ strong, .strong {
+    align-items: center;
+    justify-content: center;
+    flex: 1;
+    padding: 1rem;
+    width: 100%;
+    height: 100%;
 }
 
 // force scrollbars to appear:
 ::-webkit-scrollbar {
-  -webkit-appearance: none;
+    -webkit-appearance: none;
```

Stylelint (17.2.0)

[error] Unexpected vendor-prefixed property "-webkit-appearance" (property-no-vendor-prefix)

(property-no-vendor-prefix)